torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -521,6 +521,7 @@ void IrPrinter::handle(const kir::TensorIndex* ti) {
     }
   }
   os_ << "]";
+  os_ << " view( T" << varName(ti->view()) << " )";
```

Comment:
I think it's useful to show which TensorView this TensorIndex corresponds to.

Reply:
Would it be better to make this a little simpler as: `"T"<<ti->view()->name()`

Reply:
I'm blind, you had it right 🤦‍♂️